File: app/src/main/java/com/vmb/financialcalculator/CreditResultActivity.java
package com.vmb.financialcalculator;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

public class CreditResultActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_credit_result);

        Bundle arguments = getIntent().getExtras();
        Credit credit = (Credit) arguments.getSerializable(Credit.class.getSimpleName());

        TextView total = findViewById(R.id.textTotalCredit);
        total.setText(String.format("%,d", (long)credit.getTotal()));

        TextView interest_charge = findViewById(R.id.textInterestChargeCredit);
        interest_charge.setText(String.format("%,d", (long)credit.getInterest_charge()));

        TextView monthly_installment = findViewById(R.id.textMonthlyInstallmentCredit);
        monthly_installment.setText(String.format("%,d", (long)credit.getMonthly_installment()));
    }
}

File: app/src/main/java/com/vmb/financialcalculator/CreditResultActivity.kt
package com.vmb.financialcalculator

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.TextView

class CreditResultActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_credit_result)
        val arguments = intent.extras
        val credit = arguments!!.getSerializable(Credit::class.java.simpleName) as Credit?
        val total = findViewById<TextView>(R.id.textTotalCredit)
        total.text = String.format("%,d", credit!!.total.toLong())
        val interest_charge = findViewById<TextView>(R.id.textInterestChargeCredit)
        interest_charge.text = String.format("%,d", credit.interest_charge.toLong())
        val monthly_installment = findViewById<TextView>(R.id.textMonthlyInstallmentCredit)
        monthly_installment.text = String.format("%,d", credit.monthly_installment.toLong())
    }
}